--- vm_image_util.sh.bkp	2015-10-26 16:49:46.862168809 -0700
+++ vm_image_util.sh	2015-11-29 15:15:28.004959072 -0800
@@ -38,6 +38,8 @@
     cloudsigma
     packet
     interoute
+    vmturbo_hyperv
+    vmturbo_openstack
 )
 
 #list of oem package names, minus the oem- prefix
@@ -59,6 +61,7 @@
     cloudsigma
     packet
     interoute
+    vmturbo
 )
 
 # Set at runtime to one of the above types
@@ -267,6 +270,14 @@
 IMG_interoute_OEM_PACKAGE=oem-interoute
 IMG_interoute_BUNDLE_FORMAT=ova
 
+## vmturbo_hyperv
+IMG_vmturbo_hyperv_OEM_PACKAGE=oem-vmturbo
+IMG_vmturbo_hyperv_DISK_FORMAT=vhd
+
+## vmturbo_openstack
+IMG_vmturbo_openstack_OEM_PACKAGE=oem-vmturbo
+IMG_vmturbo_openstack_DISK_FORMAT=qcow2
+
 ###########################################################
 
 # Validate and set the vm type to use for the rest of the functions
@@ -421,6 +432,9 @@
         --root-deps=rdeps --usepkg --quiet "${oem_pkg}"
     sudo rsync -a "${oem_tmp}/usr/share/oem/" "${VM_TMP_ROOT}/usr/share/oem/"
     sudo rm -rf "${oem_tmp}"
+
+    echo "--- LISTING OEM PARTITION ---"
+    ls -l "${VM_TMP_ROOT}/usr/share/oem/"
 }
 
 # Any other tweaks required?
